src/main/java/fact/io/zfits/BlockHeader.java

```diff
@@ -15,7 +15,7 @@
 		
 		private final short id;
 		// using int and casting to short saves us the work to write '(short)x' everywhere above
-		private Prozessor(int id) {
+		Prozessor(int id) {
```

Comment:
It's really small and nothing really important, but could we remove the german naming here?